integration/apps_test.go

```diff
@@ -514,6 +514,32 @@ var _ = suite("apps/update", func(t *testing.T, when spec.G, it spec.S) {
 			expect.Equal(expectedOutput, strings.TrimSpace(string(output)))
 		})
 	})
+	when("the create flag is passed", func() {
```

Comment:
This should result in basic coverage of testing the flag behaviour but does not test the app creation path.